chainstate/src/interface/chainstate_interface_impl.rs

```diff
@@ -340,4 +342,44 @@ impl<S: BlockchainStorage> ChainstateInterface for ChainstateInterfaceImpl<S> {
             .get_block_id_tree_as_list()
             .map_err(ChainstateError::FailedToReadProperty)
     }
+
+    fn import_bootstrap_stream<'a>(
+        &mut self,
+        reader: std::io::BufReader<Box<dyn std::io::Read + Send + 'a>>,
+    ) -> Result<(), ChainstateError> {
+        let magic_bytes = self.chainstate.chain_config().magic_bytes().to_vec();
+
+        let mut reader = reader;
```

Comment:
You prefer not to write `mut reader: ...` in the funtion signature, right?..

Reply:
Not really... it's weird on so many levels, but I'm not fully decided.